<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pic</class>
 <widget class="QWidget" name="pic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pic</string>
  </property>
  <widget class="QLabel" name="picBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
